kiwi/builder/live.py

```diff
@@ -297,8 +297,14 @@ def create(self):
 
     def _create_dracut_live_iso_config(self):
         live_config_file = self.root_dir + '/etc/dracut.conf.d/02-livecd.conf'
+        omit_modules = [
+            'kiwi-dump', 'kiwi-overlay', 'kiwi-repart', 'kiwi-lib'
+        ]
         live_config = [
```

Comment:
@schaefi, just curiosity, why we need now to omit those modules? I assume they were not created at the time of writing this method, is that the case?

Reply:
The reason is if e.g by accident someone installs another kiwi dracut module e.g dracut-kiwi-oem-repart this module will be also installed and hooks itself into the premount stage. As we create a generic dracut module (hostonly=false) all installed modules will be used. Therefore I omit the ones where I know would conflict with kiwi-live and/or dmsquash-live

Reply:
Oh, I see, thanks, good to know